specs/capella/beacon-chain.md

```diff
@@ -301,9 +321,11 @@ def process_epoch(state: BeaconState) -> None:
     process_participation_flag_updates(state)
     process_sync_committee_updates(state)
     process_full_withdrawals(state)  # [New in Capella]
+    process_partial_withdrawals(state)  # [New in Capella]
```

Comment:
I get the consistency argument, but we're not processing partial withdrawals so much as generating them.  Not sure this matters in any real sense, and as I say having `process_*` keeps things consistent if not totally accurate.  So feel free to ignore this comment :)